src/components/settings/ContractVersion/index.tsx

```diff
@@ -42,6 +45,27 @@ export const ContractVersion = ({ isGranted }: { isGranted: boolean }) => {
           Unsupported contract
         </Typography>
       )}
+      <>
+        <br></br>
+        <br></br>
+        {isUpdated! ? (
+          <Typography variant="body1" fontWeight={400}>
+            <Typography variant="body1" fontWeight={400}>
+              ✅ Latest version
+            </Typography>
+          </Typography>
+        ) : (
+          <Tooltip
+            title="Update now to take advantage of new features and the highest security standards available.
+You will need to confirm this update just like any other transaction."
+            placement="right-end"
+          >
+            <Link sx={{ fontSize: '14px' }} href="">
+              ℹ️ Why should I upgrade?
```

Comment:
For the info icon, we've used this in other places:
```
import InfoIcon from '@/public/images/notifications/info.svg'

        <SvgIcon
          component={InfoIcon}
          inheritViewBox
          fontSize="small"
          color="border"
        />

```

Reply:
yes, changed that as well. thanks.